src/main/java/frc/robot/common/Odometry.java

```diff
@@ -48,6 +52,17 @@ public void update(SwerveModulePosition[] newPositions) {
 
         // Update the pose
         pose = odometry.update(gyroAngle, newPositions);
+        // System.out.println(this.getPose().getRotation().getDegrees());
+
+        if (limelight.getIsDetecting()){
+            if(limelight.getTargetRotation() == 180)
```

Comment:
is this supposed to be `==`? Seems unlikely it will be exactly 180

Reply:
I think their limelight subsystem's target rotation is set to only 0 or 180 based on which alliance the robot is. Maybe you all want to switch this to an enum instead?

Reply:
It's based partially on alliance and partially on which apriltag is being picked up. See LimelightSubsystem.java 46-58 for that code.

We could use an enum, but wouldn't that be more inefficient? Then any time we want to use the targetRotation for its actual purpose (rotating), we'd need another if statement.

Reply:
You can add additional constants to enum members as a field that you pass into the member definition like a function argument. Then, you can access the field with a getter method.

The enum itself is just to improve readability. Check out these docs:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html